tfe/resource_tfe_project.go

```diff
@@ -0,0 +1,109 @@
+package tfe
+
+import (
+	"context"
+	"errors"
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+)
+
+func resourceTFEProject() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceTFEProjectCreate,
+		ReadContext:   resourceTFEProjectRead,
+		UpdateContext: resourceTFEProjectUpdate,
+		DeleteContext: resourceTFEProjectDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"organization": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceTFEProjectCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	tfeClient := meta.(*tfe.Client)
+
+	organizationName := d.Get("organization").(string)
+	name := d.Get("name").(string)
+
+	options := tfe.ProjectCreateOptions{
+		Name: name,
+	}
+
+	log.Printf("[DEBUG] Create new project: %s", name)
+	project, err := tfeClient.Projects.Create(ctx, organizationName, options)
+	if err != nil {
+		return diag.Errorf("Error creating the new project %s: %v", name, err)
+	}
+
+	d.SetId(project.ID)
+
+	return resourceTFEProjectUpdate(ctx, d, meta)
+}
+
+func resourceTFEProjectRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	tfeClient := meta.(*tfe.Client)
+
+	log.Printf("[DEBUG] Read configuration of project: %s", d.Id())
+	project, err := tfeClient.Projects.Read(ctx, d.Id())
+	if err != nil {
+		if errors.Is(err, tfe.ErrResourceNotFound) {
+			log.Printf("[DEBUG] Project %s does no longer exist", d.Id())
```

Comment:
```suggestion
			log.Printf("[DEBUG] Project %s no longer exist", d.Id())
```

Reply:
I got that from over-zealous copy-paste, so I went ahead and updated all the instances of that typo. There were several; seems like I'm not the first one!